sql/pg_duckdb--0.0.1.sql

```diff
@@ -232,12 +257,15 @@ REVOKE ALL ON tables FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION install_extension(extension_name TEXT) RETURNS bool
     LANGUAGE C AS 'MODULE_PATHNAME', 'install_extension';
+REVOKE ALL ON FUNCTION install_extension(TEXT) FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION raw_query(query TEXT) RETURNS void
     LANGUAGE C AS 'MODULE_PATHNAME', 'pgduckdb_raw_query';
+REVOKE ALL ON FUNCTION raw_query(TEXT) FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION cache(object_path TEXT, type TEXT) RETURNS bool
     LANGUAGE C AS 'MODULE_PATHNAME', 'cache';
+REVOKE ALL ON FUNCTION cache(TEXT, TEXT) FROM PUBLIC;
```

Comment:
is this being blocked because its parameters do not get escaped?

Reply:
Yeah. This definitely doesn't seem safe:

```
	auto cache_object_query =
	    duckdb::StringUtil::Format("SELECT 1 FROM %s('%s');", object_type_fun, object_path.c_str());
```

Reply:
😬

Reply:
To be clear, I think we can probably get these functions SQL injection safe fairly easily. I just didn't want to batch even more into this already fairly big and complex PR.